libbeat/outputs/kafka/docs/kafka.asciidoc

```diff
@@ -79,12 +71,28 @@ See <<kafka-compatibility>> for information on supported versions.
 ===== `username`
 
 The username for connecting to Kafka. If username is configured, the password
-must be configured as well. Only SASL/PLAIN is supported.
+must be configured as well.
 
 ===== `password`
 
 The password for connecting to Kafka.
 
+===== `sasl.mechanism`
```

Comment:
Shall we add the new setting to the reference config file as well?

Reply:
Added.